### ISSUE TITLE
Refactor automated a11y scans

### ISSUE THREAD

**Issue by pboeder** [porscheofficial/porscheofficial.github.io#17]
# ✨ Refactor: A11y scans

The A11y scan runs automatically after each push or manually.
That should be changed, because we need to await the deployment process, before running the a11y tests.
Additionally the paths that shall be tested need to be listed in `tests/a11y.spec.ts`. With the new blog feature, that should be automated, to include the latest blog posts as well.

## Acceptance criteria

* A11y scans should include all deployed paths automatically
* The A11y scans should be run after each successful deployment

**Comment by itrich** [comment]
As for now, I have disabled the A11y tests for `svg` elements since https://github.com/Niels-IO/next-image-export-optimizer is not creating a unique id for every blur filter. This needs to be addressed upstream.

See https://github.com/porscheofficial/porscheofficial.github.io/commit/9a124e4103c60c23572d12ed3ff25786f06c010f#diff-f5fca1f01ef788e9a4af430797533798b8352aef23031a49446e0beae8361112R17
